README.md

```diff
@@ -1,3 +1,5 @@
+<img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
```

Comment:
OpenSearch uses a different asset. 
Do you think having a consistent asset makes more sense?
https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg
Ref: https://raw.githubusercontent.com/opensearch-project/OpenSearch/main/README.md

Reply:
This themed SVG uses the "default" and "dark" modes from [OpenSearch Branding page](https://opensearch.org/brand.html) in one file to show the appropriate one to the users based on their browser's theme.

Reply:
Got it. Thanks for that info. It means we have to change OpenSearch repo with this new asset.